python_ta/checkers/inconsistent_or_missing_returns_checker.py

```diff
@@ -71,6 +82,12 @@ def _check_return_statements(self, node) -> None:
         if has_return_annotation or has_return_value:
             for block, statement in return_statements.items():
                 if statement is None:
+                    # ignore unfeasible edges for missing return if z3 option is on
+                    if self.linter.config.z3 and all(
```

Comment:
This is good, but let's do some simplification here. Please add a [property](https://docs.python.org/3/library/functions.html#property) to `CFGBlock` called `is_feasible`, and then use that here (and throughout your changes, where appropriate)